types/react-native-push-notification/index.d.ts

```diff
@@ -84,6 +84,7 @@ export class PushNotificationObject {
     id?: string | number | undefined;
     title?: string | undefined;
     message: string;
+	picture?: string;
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/react-native-push-notification/index.d.ts;
  Line: 87;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.